README.md

```diff
@@ -1,115 +1 @@
-# venn [![Build Status](https://travis-ci.org/bitoiu/venn.png)](https://travis-ci.org/bitoiu/venn) [![NPM version](https://badge.fury.io/js/venn.png)](http://badge.fury.io/js/venn)
```

Comment:
> I agree with this

@nigelabbott also said so on a 1:1 meeting.

*Note: Tim's requirement of tagging other people, albeit very informally*